templates/company-board/README.md

```diff
@@ -71,3 +73,16 @@ The network details will be automatically selected by the `arapp.json`'s environ
 | Board Token Manager | BURN                  | Share Voting        | Share Voting  |
 | Share Token Manager | MINT                  | Share Voting        | Share Voting  |
 | Share Token Manager | BURN                  | Share Voting        | Share Voting  |
+
+## Permissions for the Payroll app
```

Comment:
```suggestion
### Additional permissions if the Payroll app is installed
```

I think it makes more sense as a sub-section in permissions, since these are optional details.